docs/examples/Sensor_Platform_Simulation.py

```diff
@@ -0,0 +1,380 @@
+
+# First we need to configure a radar
+
+# Generate a radar sensor with a suitable measurement accuracy
+noise_covar = CovarianceMatrix(np.array(np.diag([np.deg2rad(5)**2,#
+                                                 np.deg2rad(0.14)**2,
+                                                 25**2])))
+# this radar measures range with an accuracy of +/- 25m, and elevation accuracy +/- 5
+# degrees and bearing accuracy of +/- 0.14 degrees
+
+# The radar needs to be informed of where x, y, and z are in the target state space
+radar_mapping = (0, 2, 4)
+
+# Instantiate the rotating radar
+radar = RadarRangeBearingElevation(ndim_state=6,
+                                   position_mapping=radar_mapping,
+                                   noise_covar=noise_covar)
+# %%
+# Attach the sensor to the platform
+# ---------------------------------
+# Now that we have created our radar sensor we need to mount the sensor onto the platform we have previously created.
+#
+# Sensors can be mounted with two additional parameters; the mounting offset and rotation offset.
+#
+# The mounting offset:
+#     * defines how the sensors position is offset from the platform,
+#     * defaults to a position offset of zero.
+# The rotation_offset:
+#     * defines the sensors orientation relative to that of the platform,
+#     * defaults to a zero orientation offset.
+#
+# The default assumption is that the sensor is located at the center point of the platform and orientated to align with
+# the platform body. In this example we are happy to use the default assumptions and therefore the sensor can be added.
+platform.add_sensor(radar)
+
+# %%
+# As before we can query the platform to demonstrate that it has a sensor mounted:
+platform.sensors
+
+# %%
+# You will notice that platform.sensors returns a list which contains our single sensor. This hints at the multi-sensor
+# platform functionality which is shown in a subsequent example.
+#
+# We can also check to ensure that the default mounting_offsets have been applied:
+platform.mounting_offsets
+
+# %%
+# And that the rotation_offsets have been applied:
+platform.rotation_offsets
+
+# %%
+# Building a simulation
+# ---------------------
+# Now that we have created a sensor platform we need to build a simulation which generates targets for the sensor to
+# detect and track. For this example we are going to use a :class:`~.MultiTargetGroundTruthSimulator`. this simulator
+# enables multiple ground truth targets to be created based on a number of user defined parameters.
+#
+# In this example targets are initiated with values based upon a mean state and a covariance, using a Gaussian
+# assumption. This is done by creating a :class:`~.GaussianState` object which describes the distribution from which we
+# want our targets to be drawn from. For this example targets will be drawn from:
+#   * :math:`x` is Gaussian distributed around the platform location with variance of 2km
```

Comment:
This could probably be written more compactly as :math:`\mathcal{N}(\mathbf{x}_p, \diag([2,2,0.1,50,50,1])` (or similar)

Units should be roman, not italic (\mathrm{})

Reply:
Tried the compact solution and it looks messy so reverted to original structure